usr/src/cmd/zfs/zfs_main.c

```diff
@@ -204,6 +206,7 @@ static zfs_command_t command_table[] = {
 	{ "holds",	zfs_do_holds,		HELP_HOLDS		},
 	{ "release",	zfs_do_release,		HELP_RELEASE		},
 	{ "diff",	zfs_do_diff,		HELP_DIFF		},
+	{ "key",	zfs_do_crypto,		HELP_CRYPTO		},
```

Comment:
Unless there's a good reason, these 3 strings should have the same "x":
`"x", zfs_do_x, HELP_X`

It looks like the indentation is wrong here, but maybe that's just github.